### PR TITLE
Fix typography

### PR DESCRIPTION
**Pull request by viliam-durina**
- Replace `text :text` with `text: text`
- Remove space before "!" and "."
- Replace "!!!" with "!" (I would even consider removing "!" from most exceptions thrown, no need to scream at the user)
- Some typos

### CODE REVIEW

File: hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java [hunk 1]
```diff
@@ -422,7 +422,7 @@ private void addNearCacheInvalidateListener() {
             invalidationListenerId = registerListener(createNearCacheInvalidationListenerCodec(), handler);
         } catch (Exception e) {
             ILogger logger = getContext().getLoggingService().getLogger(ClientReplicatedMapProxy.class);
-            logger.severe("-----------------\n Near Cache is not initialized!!! \n-----------------", e);
+            logger.severe("-----------------\n Near Cache is not initialized! \n-----------------", e);
```
Comment (Donnerbart) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java]: No space needed before the second `\n` (and I would also remove it after the first `\n`)

File: hazelcast-client/src/main/java/com/hazelcast/client/proxy/NearCachedClientMapProxy.java [hunk 2]
```diff
@@ -481,7 +481,7 @@ public void addNearCacheInvalidationListener(EventHandler handler) {
             invalidationListenerId = registerListener(createNearCacheEntryListenerCodec(), handler);
         } catch (Exception e) {
             ILogger logger = getContext().getLoggingService().getLogger(getClass());
-            logger.severe("-----------------\n Near Cache is not initialized!!! \n-----------------", e);
+            logger.severe("-----------------\n Near Cache is not initialized! \n-----------------", e);
```
Comment (Donnerbart) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/NearCachedClientMapProxy.java]: Same here, superfluous spaces.

File: hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/xa/XATransactionContextProxy.java [hunk 3]
```diff
@@ -71,24 +71,24 @@ public XATransactionContextProxy(ClientTransactionManagerServiceImpl transaction
...
     @Override
     public void commitTransaction() throws TransactionException {
-        throw new UnsupportedOperationException("XA Transaction cannot be committed manually!!!");
+        throw new UnsupportedOperationException("XA Transaction cannot be committed manually!");
     }
 
     @Override
     public void rollbackTransaction() {
-        throw new UnsupportedOperationException("XA Transaction cannot be rollbacked manually!!!");
+        throw new UnsupportedOperationException("XA Transaction cannot be rolled back manually!");
```
Comment (Donnerbart) [inline hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/xa/XATransactionContextProxy.java]: rollbacked

**Comment by viliam-durina** [comment]
What do you think about removing exclamation marks from exceptions? There are 485 of them :)

**Comment by viliam-durina** [comment]
verify

**Pull request merged**